### PR TITLE
Structure_oM: Renaming master/slave to primary/secondary

### LINKED ISSUE #921: Structure_oM: rename RigidLink master/slave nodes
While it's only a tiny brick in the gargantuan wall that is structural racism, it seems like a good day to quietly re-name MasterNode and SlaveNodes to something without the pain embedded in those terms.

I suggest Planet/Satellite or Sun/Planet, or maybe Base/Remote

The important thing to illustrate with these names is that the head/sun/base owns the coordinate system that defines the constraint between it and the other nodes.

### PR DESCRIPTION
**Pull request by JosefTaylor**
THOW OFF THE CHAINS OF OPPRESSION!

### Issues addressed by this PR

Closes #921 

renamed master/slave to primary/secondary

### Test files

https://burohappold.sharepoint.com/:u:/r/sites/BHoM/02_Current/12_Scripts/01_Test%20Scripts/BHoM/Structure_oM/Issue921-RigidLink%20property%20rename/Test%20Rigid%20Links%20-%20multi%20adapter.gh?csf=1&web=1&e=iDXtGs

### CODE REVIEW

**Comment by IsakNaslundBh** [comment]
Looks great. Before pushing it any more though, would be good to get https://github.com/BHoM/BHoM/pull/959 closed out, as both of these PRs will affect Versioning.

File: Structure_oM/Versioning_33.json [hunk 1]
```diff
@@ -17,8 +17,12 @@
   },
   "Property": {
     "ToNew": {
+    "BH.oM.Structure.Elements.RigidLink.PrimaryNode": "BH.oM.Structure.Elements.RigidLink.MasterNode"
```
Comment (IsakNaslundBh) [inline Structure_oM/Versioning_33.json:20]: Think the ToNew and ToOld should be switched here!

basically, ToNew is "Old" : "New" and vice versa

Comment (JosefTaylor) [reply Structure_oM/Versioning_33.json:20]: Thanks!

**Pull request merged**